### PR TITLE
doc: Remove old new-in-version tags, small fixes

### LINKED ISSUE #310: Remove old "new in" tags
#### Summary
Remove "new-in-version" tags for everything prior to 3.11

#### Why do we need this ?
Keep things up to date

#### Can you do this yourself and submit a Pull Request?
Yes

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Closes #310

#### Notes for Reviewers
Should sections that have been marked as removed in certain version be completely removed from the documentation? Example: application linking -> https://www.thethingsindustries.com/docs/integrations/adding-applications/

Should I leave **Deprecated in version xxx** sections untouched? What does deprecated mean here, that it will be removed in the next version?

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by KrishnaIyer — APPROVED** [review]
Owned files LGTM.

**Comment by johanstokking** [comment]
> Should sections that have been marked as removed in certain version be completely removed from the documentation? Example: application linking -> [thethingsindustries.com/docs/integrations/adding-applications](https://www.thethingsindustries.com/docs/integrations/adding-applications/)

Yes, this can be deleted.
 
> Should I leave **Deprecated in version xxx** sections untouched? What does deprecated mean here, that it will be removed in the next version?

Deprecated doesn't mean removed. It may still be available for compatibility purposes, and should therefore be documented still. So let's handle that on a case by case basis.

File: doc/content/integrations/aws-iot/_index.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ The recommended way to integrate with AWS IoT is by using the **Default Integrat
 
 [Learn how to set up the default AWS IoT integration]({{< relref default >}})
 
-## Application Server Telemetry {{< distributions "AWS Launcher" >}} {{< deprecated-in-version "3.11.1" >}} {#application-server-telemetry}
+## Application Server Telemetry {{< distributions "AWS Launcher" >}} {{< removed-in-version "3.12.0" >}} {#application-server-telemetry}
```
Comment (johanstokking) [inline doc/content/integrations/aws-iot/_index.md:25]: Revert, it isn't removed but deprecated in 3.11.1

File: doc/content/integrations/aws-iot/application-server-telemetry/_index.md [hunk 2]
```diff
@@ -2,12 +2,11 @@
 title: "Application Server Telemetry"
 description: ""
 distributions: "AWS Launcher"
-deprecated_in_version: "3.11.1"
 ---
 
 {{% tts %}} supports publishing of uplink messages directly to the AWS IoT suite.
 
-{{< warning >}} This integration will be removed in a next version. Please switch to the [Default Integration]({{< relref "../default" >}}) instead. {{</ warning >}}
+{{< warning >}} This integration has been removed in version `3.12.0`. Please use to the [Default Integration]({{< relref "../default" >}}) instead. {{</ warning >}}
```
Comment (johanstokking) [inline doc/content/integrations/aws-iot/application-server-telemetry/_index.md:9]: Revert

**Review by johanstokking — APPROVED** [review]

**Comment by benolayinka** [comment]
There are a few more

```
Searching 559 files for "{{< new-in-version"

/Users/ben/lorawan-stack-docs/doc/content/gateways/semtech-udp-packet-forwarder/_index.md:
   23  {{< tabs/tab "Console" >}}
   24  
   25: ## Download Configuration in the Console {{< new-in-version "3.10" >}} {#download-configuration-in-the-console}
   26  
   27  To download a `global_conf.json` file for your gateway, open the Gateway overview page in the console. Click the **Download global_conf.json** button to download the file.

/Users/ben/lorawan-stack-docs/doc/content/getting-started/aws/ecs/deployment/_index.md:
  152  For multi-tenant deployments that use tenant billing through Stripe (see the [Stripe Billing reference]({{< ref "/reference/stripe" >}})), you need to configure the Stripe API key (starting with `sk_`) and Stripe endpoint secret key (starting with `whsec_`). It is possible to add or update this in the future.
  153  
  154: For the **Gateway Secrets Encryption Key Value** parameter {{< new-in-version "3.10" >}}, provide an AES-128 Key in Base64. The following command can be used to randomly generate one:
  155  
  156  ```bash
  ...
  248  >We recommend to start with 2 instances of each service. For some services it is not possible (yet) to deploy more than one instance.
  249  
  250: ## Gateway Configuration Service (optional) {{< new-in-version "3.11.3" >}} {#gateway-configuration-service}
  251  
  252  The `5-3d-ecs-gcs-service` template creates a stand-alone Gateway Configuration Server (GCS) service with a separate console.

/Users/ben/lorawan-stack-docs/doc/content/getting-started/aws/ecs/monitoring/_index.md:
   61  For some of these metrics, there are also variants with `:avg` and `:stddev` suffixes that can be used for anomaly detection, and are therefore also used in alerting rules.
   62  
   63: For the Packet Broker Agent: {{< new-in-version "3.11.2" >}}
   64  
   65  - `ttn_lw_pba_uplink_received_rate` record the rate of uplink messages received from Packet Broker (as a home network).

/Users/ben/lorawan-stack-docs/doc/content/getting-started/migrating/import-devices.md:
   13  {{< tabs/tab "Console" >}}
   14  
   15: ### Import devices via the Console {{< new-in-version "3.9.3" >}} {#import-devices-via-the-console}
   16  
   17  Open the application you created and click the button **Import end devices**

/Users/ben/lorawan-stack-docs/doc/content/integrations/aws-iot/default/architecture.md:
   16  - AWS Lambda functions to create the thing type and configure the integration as pub/sub in {{% tts %}}
   17  - AWS Lambda functions for claiming and creating devices, and for handling uplink and downlink messages
   18: - Secret with key encryption key (KEK) to leverage LoRaWAN end-to-end encryption {{< new-in-version "3.10.0" >}}
   19  - IoT Core rules to trigger the Lambda functions based on topics and attributes
   20  

/Users/ben/lorawan-stack-docs/doc/content/integrations/aws-iot/default/deployment-guide.md:
   69    - When using **The Things Stack Enterprise**, enter your cluster address
   70    - When using **The Things Network**, select the community cluster from the dropdown
   71: - **Enable End-to-End Encryption** {{< new-in-version "3.10.0" >}} {{< distributions "Cloud" >}}: If enabled, the AppSKey is delivered as encrypted from Join Server to your AWS Account, so the AppSKey will not be exposed to the network layer. Also, your AWS solution needs to handle binary payload as the underlying network cannot run payload encoding and decoding functions.
   72  - **Application ID**: The application ID for which you configure the integration.
   73  - **Application API Key**: The application API key that you generated before.

/Users/ben/lorawan-stack-docs/doc/content/integrations/storage/retrieve.md:
   38  </details>
   39  
   40: {{< new-in-version "3.11" >}} Use field masks to specify a subset of fields that should be returned by the API. For example, to retrieve the decoded payload field only, set the field mask as shown below. Note that the end device identifiers and the timestamp are always included in the message.
   41  
   42  ```bash

/Users/ben/lorawan-stack-docs/doc/content/reference/configuration/identity-server.md:
  115  By default, users can register their own user accounts. User accounts can also be registered by admin users in the network. The user registration process can be customized by requiring approval by admin users, requiring email validation or by requiring new users to be invited by existing users.
  116  
  117: - `is.user-registration.enabled`: Enable user registration. If user registration is disabled, admin users can still create users. {{< new-in-version "3.11" >}}
  118  - `is.user-registration.admin-approval.required`: Require admin approval for new users
  119  - `is.user-registration.contact-info-validation.required`: Require contact info validation for new users

/Users/ben/lorawan-stack-docs/doc/content/reference/configuration/packet-broker-agent.md:
    4  ---
    5  
    6: ## Registration Options {{< new-in-version "3.12.0" >}}
    7  
    8  - `pba.registration.name`: Friendly name to register with Packet Broker
    .
   13  ## Connection Options
   14  
   15: - `pba.iam-address`: Address of Packet Broker IAM {{< new-in-version "3.12.0" >}}
   16: - `pba.control-plane-address`: Address of Packet Broker Control Plane {{< new-in-version "3.12.0" >}}
   17  - `pba.data-plane-address`: Address of Packet Broker Data Plane. See [Packet Broker Clients](https://github.com/packetbroker/pb) for available cluster addresses
   18  - `pba.insecure`: Connect without using TLS (only for test environments)
   ..
   34  - `pba.forwarder.worker-pool.limit`: Limit of active workers concurrently forwarding uplink messages and processing downlink message
   35  - `pba.forwarder.token-key`: AES 128 or 256-bit key for encrypting uplink token
   36: - `pba.forwarder.include-gateway-eui`: Indicates whether to include the gateway EUI in forwarded uplink messages {{< new-in-version "3.12.0" >}}
   37: - `pba.forwarder.include-gateway-id`: Indicates whether to include the gateway ID (plain or hashed) in forwarded uplink messages {{< new-in-version "3.12.0" >}}
   38: - `pba.forwarder.hash-gateway-id`: Indicates whether the forwarded gateway ID should be hashed. This setting is ineffective if `include-gateway-id` is `false` {{< new-in-version "3.12.0" >}}
   39  
   40  ## Home Network Options

/Users/ben/lorawan-stack-docs/doc/content/reference/configuration/the-things-stack.md:
   68  If {{% tts %}} is deployed behind a reverse proxy that does not use a private network IP address (in `127.0.0.0/8`, `10.0.0.0/8`, `100.64.0.0/10`, `172.16.0.0/12` or `192.168.0.0/16`), its IP address range needs to be configured in order for {{% tts %}} to trust it.
   69  
   70: - `grpc.trusted-proxies`: CIDRs of trusted reverse proxies. {{< new-in-version "3.10.2" >}}

```

**Review by adriansmares — APPROVED** [review]

**Review by htdvisser — APPROVED** [review]

**Pull request merged**